test/Data/Geospatial/Internal/CRSTests.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Data.Geospatial.Internal.CRSTests where
```

Comment:
Thanks for removing doctest!

It would have been quite a time consuming exercise